### DISCUSSION TITLE
Habit tracker: un-archive?

### DISCUSSION THREAD

**Discussion by czuu42** [iSoron/uhabits#854] [Help & FAQ] [answered]
Hello! I love the habit tracker app.

I'm wondering if there's a way to un-archive habits? I had a habit that I stopped practicing a few months ago, and now I want to reincorporate it into my routine. I realize I could create a new habit in the app, but this seems redundant.

If this is not already a feature, I'd like to request it.

Thanks!
Hope

**Comment by hiqua** [comment, marked as answer]
Yes there is, untick "Hide archived" (in the menu with the three dashes, the middle icon), select the habit and click "Unarchive".
